docs/getting-started/index.html

```diff
@@ -4,19 +4,55 @@
 group: "navigation"
 ---
 <div>
-	<h2>Getting Started</h2>
-	<p>Below you will find various materials to give you an introduction to the W3C Web of Things and its technology.</p>
+	<h2>Introduction</h2>
+	<p>The W3C Web of Things (WoT) seeks to counter the fragmentation of the IoT, making it much easier to create 
+		applications without the need to master the disparate variety of IoT technologies and standards. 
+		To realize this, WoT specifies some principles and technology building blocks that can be used to 
```

Comment:
To realize this, WoT specifies some principles and technology building blocks that were derived from requirements addressing these use cases. These building blocks can be used to